pkg/charset/CharsetManager_test.go

```diff
@@ -24,11 +24,11 @@ func TestDecodeFromCP866(t *testing.T) {
 
 }
 
-func TestEncodeToCP866(t *testing.T) {
+func TestEncodeToCP850(t *testing.T) {
 
 	charsetManager := NewCharsetManager(nil)
 
-	var msgBody string = "Привет, мир!"
+	var msgBody string = "Hello World!"
```

Comment:
Actually I does not well known about using language in CP850
But unittest should use CP850 specific char to check runes like 'ü' or 'å'.
Test with Latin words is not well check charset translation.
Could you write some message on specific language to check that special char translation.
Result byte array should contain Unicode representation of you message on decoding,
and CP850 charset on encoding.